scavenger-agent-java/src/integrationTest/java/integrationTest/javaagent/ConfigTest.java

```diff
@@ -0,0 +1,67 @@
+package integrationTest.javaagent;
+
+import static integrationTest.util.AgentLogAssertionUtil.assertDisabled;
+import static integrationTest.util.AgentLogAssertionUtil.assertSampleAppOutput;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+
+import java.util.Properties;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import integrationTest.support.AgentIntegrationTestContextProvider;
+import integrationTest.support.AgentRunner;
+
+@ExtendWith(AgentIntegrationTestContextProvider.class)
+@DisplayName("config test")
+public class ConfigTest {
+
+    @TestTemplate
+    @DisplayName("if no configuration is found")
+    void noConfig(AgentRunner agentRunner) throws Exception {
+        // given
+        agentRunner.setConfigFilePath("");
+
+        // when
+        String stdout = agentRunner.call();
+
+        // then
+        assertThat(stdout, containsString("Configuration file is not found"));
```

Comment:
I think it's better to use assertj way.
Like... ```assertThat(actual).containsString("Configuration file is not found")```

Reply:
fixed in [08d3010](https://github.com/naver/scavenger/pull/71/commits/08d3010fbf4dcd732fc8c68e73fd18de3d4d9209).